styx-service-common/src/main/java/com/spotify/styx/storage/DatastoreStorageTransaction.java

```diff
@@ -232,7 +234,9 @@ public Backfill store(Backfill backfill) throws IOException {
         .set(PROPERTY_NEXT_TRIGGER, instantToTimestamp(backfill.nextTrigger()))
         .set(PROPERTY_ALL_TRIGGERED, backfill.allTriggered())
         .set(PROPERTY_HALTED, backfill.halted())
-        .set(PROPERTY_REVERSE, backfill.reverse());
+        .set(PROPERTY_REVERSE, backfill.reverse())
```

Comment:
What would happen when BackfillTriggerManager reads active backfill created before this PR is deployed and progresses it?